Snake.js

```diff
@@ -199,32 +196,27 @@ Snake.prototype.createGeometry = function( geometry  , numOf ){
       normals[ index * 3  + 6 ] = n3.x;
       normals[ index * 3  + 7 ] = n3.y;
       normals[ index * 3  + 8 ] = n3.z;
-    
+   
+    } 
+        
+  for( var i = 0; i < numOf; i++ ){
       var y = (Math.floor( i / lookupSize ))/ lookupSize;
       var x = (i - ( (Math.floor( i / lookupSize )) * lookupSize )) / lookupSize;
     
        
       var a = .5 / lookupSize;
 
-      lookups[ index * 2 + 0 ] = x + a;
-      lookups[ index * 2 + 1 ] = y + a;
+      lookups[ i * 2 + 0 ] = x + a;
+      lookups[ i * 2 + 1 ] = y + a;
       
-      lookups[ index * 2 + 2 ] = x + a;
-      lookups[ index * 2 + 3 ] = y + a;
-
-      lookups[ index * 2 + 4 ] = x + a;
-      lookups[ index * 2 + 5 ] = y + a;
-
```

Comment:
This means the instanced data only needs 1 copy; rather than a copy per vertex * mesh